gallery_dl/extractor/idolcomplex.py

```diff
@@ -22,7 +22,9 @@ class IdolcomplexTagExtractor(IdolcomplexExtractor,
                               sankaku.SankakuTagExtractor):
     """Extractor for images from idol.sankakucomplex.com by search-tags"""
     pattern = [r"(?:https?://)?idol\.sankakucomplex\.com"
-               r"/\?(?:[^&#]*&)*tags=([^&#]+)"]
+               r"/\?(?:[^&#]*&)*tags=([^&#]+)"
+               r"((?:[^&#]*&)*page=([^&#]+))*"
+               r"((?:[^&#]*&)*next=([^&#]+))*"]
```

Comment:
Instead of capturing every query-parameter on its own, capture the whole query string (`(?:\?([^#]*)?`) and parse it with `text.parse_query(...)`

Reply:
... OK ignore this one. What I wrote wouldn't work. Just leave it as it is.